worldengine/model/world.py

```diff
@@ -852,7 +851,67 @@ def is_chaparral(self, pos):
     #
 
     def n_actual_plates(self):
-        return self.layers['plates'].data.max() + 1
+        return self.plates.data.max() + 1
+
+    #
+    # Accessors
+    #
+
+    @property
+    def elevation(self):
+        return self.layers['elevation']
```

Comment:
I'm not sure if a line like `world.elevation = (data, thresholds)` is confusing at all - look at it without any context, as if you didn't know anything about WorldEngine. (I updated the code-example above so it is actually correct.)
You are right in that the setters aren't used too much, I just think that if you want to use properties you may want to go all the way. Having a `set_elevation()` and not a `get_elevation()` is pretty ugly in and of itself!